xml/abstract_commands.xml

```diff
@@ -94,4 +94,82 @@
         <field name="0" bits="23:0" />
     </register>
 
+    <register name="Access Memory">
+        \begin{steps}{This command gives the debugger access to memory
+            locations, including memory-mapped registers. It performs the
+            following sequence of operations:}
+        \item Copy data from the memory location specified in {\tt arg1} into the
+            {\tt arg0} portion of {\tt data}, if \Fwrite is clear and \Ftransfer is set.
+        \item Copy data from the {\tt arg0} portion of {\tt data} into the
+            memory location specified in {\tt arg1}, if \Fwrite is set and \Ftransfer is set.
+        \item Increment {\tt arg1}, if \Fpostincrement is set.
+        \item Execute the Program Buffer, if \Fpostexec is set.
+        \end{steps}
+
+        If any of these operations fail, \Fcmderr is set and none of the
```

Comment:
Are there additional cmderr types introduced for "abstract memory access" that we would need to define? Looking at SBA there are some other memory-access specific error types that aren't currently listed in `cmderr`.

Reply:
I've added a "bus error" state. We could define to command to modify arg0 to reflect a more precise error (alignment, timeout, ...) when there is a bus error. Do you think that's a good idea?